doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-security-management-permissions-con.adoc

```diff
@@ -0,0 +1,12 @@
+[id='business-central-security-management-permissions-con_{context}']
+
+= Permissions and settings
+
+A permission is an authorization granted to a user to perform actions related to a specific resource within the application. For example, a user can have following permissions:
+
+* View a page
+* Save the project
+* View a repository
+* Delete a dashboard
+
+A permission can be granted or denied and it can be global or resource specific. You can use permissions to protect access to resources and customize features within the application.
```

Comment:
You can grant or deny permission and a permission can be global or resource specific.

Reply:
Updated